chapter1.md

```diff
@@ -178,13 +181,13 @@ Python is perfectly suited to do basic calculations. Apart from addition, subtra
 - Exponentiation: `**`. This operator raises the number to its left to the power of the number to its right. For example `4**2` will give `16`.
 - Modulo: `%`. This operator returns the remainder of the division of the number to the left by the number on its right. For example `18 % 7` equals `4`.
 
-The code in the script on the right gives some examples.
+The code in the script gives some examples.
 
 `@instructions`
 Suppose you have $100, which you can invest with a 10% return each year. After one year, it's $100 \times 1.1 = 110$ dollars, and after two years it's $100 \times 1.1 \times 1.1 = 121$. Add code to calculate how much money you end up with after 7 years, and print the result.
 
 `@hint`
-After two years you have $100 \times 1.1 \times 1.1 = 100 \times 1.1^2$. How much do you have after 7 years than? Use `*` and `**`.
+After two years you have `100 * 1.1 * 1.1 = 100  * (1.1 ** 2)`. How much do you have after 7 years then?
```

Comment:
Seems like they applied some of the fixes we have in the export script!

Reply:
actually, I just realised that we didn't have a patch for this hint, and it's broken in production 😅  . On the other hand, we have a patch for this exercise instructions (to remove MathJax), but it hasn't been fixed here. So it looks like we will either have to fix these few issues ourselves or leave most of the content patches we already have in place

Reply:
that's probably why Alex fixed this and not all the others instructions/assignment containing MathJax that we are patching